src/test/unit_tests/test_s3_bucket_policy.js

```diff
@@ -1,9 +1,11 @@
 /* Copyright (C) 2016 NooBaa */
+/* eslint max-lines-per-function: ['error', 650] */
```

Comment:
Maybe we can devide all the RPC tests here to a new mocha.describe('[RPC TEST]: s3_bucket_policy' ...)

that way we will not need to disable this rule. 
One thing that we will need to do is taking the before out of the describe a maybe use `mocha.beforeEach` and also do a `mocha.afterEach` that will remove the bucket and setup.

this way we might also be able to avoid using a global object to store data between tests.